beacon-chain/rpc/eth/beacon/handlers_validator.go

```diff
@@ -49,7 +51,26 @@ func (s *Server) GetValidators(w http.ResponseWriter, r *http.Request) {
 	}
 	isFinalized := s.FinalizationFetcher.IsFinalized(ctx, blockRoot)
 
-	rawIds := r.URL.Query()["id"]
+	var req GetValidatorsRequest
+	if r.Method == http.MethodPost {
+		err = json.NewDecoder(r.Body).Decode(&req)
+		switch {
+		case err == io.EOF:
```

Comment:
Those errors case seem not to be tested.
(Not sure about the testing policy in that case.)

Reply:
done